src/assets/img/icons/fees.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="18" height="20" viewBox="0 0 18 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Typically, SVGs for icons so far had always a size of 24x24, regardless of how big the icon inside of that box is. This is to ensure they can be sized predictably and are always properly centered.

Do you think we can get a 24x24 version of this as well? (instead of 18x20)

Reply:
aaah I just got this off the designs https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=3014%3A176247

Do you know how I can get in 34*24?

Reply:
Designers can typically help with that! :D 
It's no showstopper though. Seems like things have changed with designers..